src/nwb_datajoint/common/common_metrics.py

```diff
@@ -0,0 +1,153 @@
+from pathlib import Path
+import os
+import json
+import datajoint as dj
+import spikeinterface.toolkit as st
+import numpy as np
+    
+from .common_waveforms import Waveforms
+from .common_nwbfile import AnalysisNwbfile
+
+schema = dj.schema('common_metrics')
+
+@schema
+class MetricParameters(dj.Manual):
+    definition = """
+    # Parameters for computing quality metrics of sorted units
+    metric_params_name: varchar(200)
+    ---
+    metric_params: blob
+    """
+
+    def get_metrics_list(self):
```

Comment:
This does not have to be a function since it simply returns a list. It can just be a constant variable METRICS = ['snr',' isi_violation', 'nn_isolation', 'nn_noise_overlap']